src/main.js

```diff
@@ -121,6 +121,17 @@ if ("serviceWorker" in navigator) {
 			});
 	});
 	if (Alpine.store("deviceInfo").isMobile) initialiseInstallPrompt(); // don't run install prompt on desktop
+	if (!Alpine.store("deviceInfo").isMobile){ //to show an alert to users who are not on mobile
+		window.addEventListener("load", function() {
+			alert("Kapta works best on Android mobile devices. Please visit this page on an Android mobile device to use the app.");
```

Comment:
we could abstract this into a variable to make it easier to update in future - like if you want to add this to the language prompts